set.mm

```diff
@@ -7016,7 +7016,26 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
   Logical disjunction
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
-  We continue with logical disjunction.
+  We continue with logical disjunction.  This particular section is dedicated
+  to theorems proved without ~ df-an .
+
+  All theorems from the previous section have a transscript to ` \/ ` via
+  ~ anor .  And if we consistently replace ` /\ ` occurrences even in their
+  proofs, then we can reconstruct all the results using the ` \/ ` instead of
+  ` /\ `.  That is why ` /\ ` and ` \/ ` are known as dual operations.  Either
+  is capable of developing all of the logic the other can represent.  In
+  practise ` /\ ` is often preferred, its big advantage being grouping
+  antecedents in a convenient way.
+
+  We start out in this section with no reference to ` /\ ` (and ~ df-an ) at
+  all.  We could have continued this until all theorems not using ` /\ ` in
+  their hypotheses and conclusion are proved solely by ` \/ ` , thus completely
+  demonstrating the dual nature.  But this would have been against the general
+  proof philosophy of Metamath, that utilizes all the knowledge once developed
+  in following proofs.   At some point we thus employ mixed expressions for the
+  sake of shorter proofs.  The divergion point is roughly marked by ~ orim12d .
+  The alternative proof ~ orim12dALT shows that it is available without
+  ~ df-an , but looses against powerful ` /\ ` based shortcuts.
```

Comment:
Conjunction and disjunction are indeed dual in classical logic, but here, we only use *interdefinability*. Since the choice made in set.mm is to have implication and negation as primitive notions and all the other connectives defined in terms of these, we can simply say that *definitions are eliminable* and this implies what we want, at the risk of longer proofs (this is what I wrote in the footnote of https://github.com/metamath/set.mm/pull/2723#issuecomment-1200131017).

@wlammen : do you agree to merge this PR, and I'll try to find a rephrasing of this head comment in a following PR, so as not to block the merge ?